sql/core/src/test/scala/org/apache/spark/sql/connector/AlterTableTests.scala

```diff
@@ -101,6 +101,27 @@ trait AlterTableTests extends SharedSparkSession {
     }
   }
 
+  test("AlterTable: add column with position") {
+    val t = s"${catalogAndNamespace}table_name"
+    withTable(t) {
+      sql(s"CREATE TABLE $t (id struct<x: int>) USING $v2Format")
```

Comment:
Nit: it's odd to use `id` for a struct. I think the test would be more readable using `point` if you're adding `x` and `y`.